grpc/src/main/java/com/linecorp/armeria/internal/common/grpc/GrpcStatus.java

```diff
@@ -122,7 +123,9 @@ private static Status statusFromThrowable(Throwable t) {
         if (t instanceof ClosedStreamException || t instanceof RequestTimeoutException) {
             return Status.CANCELLED.withCause(t);
         }
-        if (t instanceof UnprocessedRequestException || t instanceof IOException) {
+        if (t instanceof UnprocessedRequestException ||
+            t instanceof IOException ||
+            t instanceof FailFastException) {
```

Comment:
`CallNotPermittedException` raised by the resilience4j module has been converted to `Status.UNAVAILABLE` already because it is wrapped by `UnprocessedRequestException`.
https://github.com/line/armeria/blob/ab7488bafc2912445f026fb6d02420f134937073/resilience4j2/src/main/java/com/linecorp/armeria/resilience4j/circuitbreaker/client/Resilience4JCircuitBreakerClientHandler.java#L115-L118